code/go/0chain.net/sharder/blockstore/interface.go

```diff
@@ -0,0 +1,45 @@
+package blockstore
+
+import "github.com/go-redis/redis"
+
+var (
+	bwrecDB blockStorage
+)
+
+type (
+	blockStorage interface {
```

Comment:
Its better to name interface as `blockStorageI` or `blockStorager` rather than `blockStorage`